dev/io.openliberty.jakartaee10.internal_fat/bnd.bnd

```diff
@@ -0,0 +1,43 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+src: \
+	fat/src
+
+
+fat.project: true
+
+tested.features:\
+    jakartaee-10.0, \
+    webProfile-10.0, \
+    all_features
```

Comment:
Since don't have the other test in here, you can remove all_features.

Reply:
done